kasa/smart/modules/childdevicemodule.py

```diff
@@ -8,12 +7,4 @@ class ChildDeviceModule(SmartModule):
     """Implementation for child devices."""
 
     REQUIRED_COMPONENT = "child_device"
-
-    def query(self) -> Dict:
-        """Query to execute during the update cycle."""
-        # TODO: There is no need to fetch the component list every time,
-        #  so this should be optimized only for the init.
-        return {
-            "get_child_device_list": None,
-            "get_child_device_component_list": None,
```

Comment:
`get_child_device_component_list` might be needed back in if we want to support device pairing w/o restarts in the future.